RELEASE/scripts/autoscend/quests/optional.ash

```diff
@@ -491,6 +491,30 @@ boolean startArmorySubQuest()
 	return false;
 }
 
+boolean armorySideQuest()
+{
+	//do the quest [Lending a Hand (and a Foot)] and unlock [madeline's baking supply] store
+	if(!get_property("auto_doArmory").to_boolean())		//post setting indicating we should do this quest this ascension
+	{
+		return false;
+	}
+	startArmorySubQuest();
+	
+	if(internalQuestStatus("questM25Armorer") > -1 && internalQuestStatus("questM25Armorer") < 4)
+	{
+		set_property("choiceAdventure1061", 1);			//try to enter office
```

Comment:
should be handled in auto_choice_adv.ash

Reply:
fixed